src/models/secretCodes.js

```diff
@@ -0,0 +1,29 @@
+import { BaseModel } from './base';
+
+class SecretCodeModel extends BaseModel {
+  constructor() {
+    super('secret_codes');
+  }
+
+  async addSecretCode(values) {
+    const query = `
+      INSERTO INTO ${this.table}(code)
```

Comment:
Será que nao deveria ser assim, visto que o VALUES espera 3 valores?
```suggestion
      INSERTO INTO ${this.table}(code, user_id, expiration_date)
```